src/drivers/telemetry/bst/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ px4_add_module(
 	COMPILE_FLAGS
 	SRCS
 		bst.cpp
+		bst_params.c
```

Comment:
Not actually compiled (confusing I know).

Reply:
there are 12 more place that its on the CMakeLists.txt. Do you want me to remove them?

Reply:
Just here is fine. At some point we'll do something so that these are no longer c files (yaml perhaps).